### PR TITLE
Various fixes to bring the example app up to speed

### PR DESCRIPTION
**Pull request by demianbrecht**
We should probably have an automated test in the examples app.

### CODE REVIEW

**Comment by bgrant** [comment]
`formatcheck` (Black) is failing

File: django_declarative_apis/machinery/__init__.py [hunk 1]
```diff
@@ -183,13 +183,14 @@ def get_response(self):
                 else:
                     raise HttpStatusCode(data)
             else:
+                try:
+                    x_expand = self.bound_endpoint.request.META.get("HTTP_X_EXPAND")
+                except:
```
Comment (bgrant) [inline django_declarative_apis/machinery/__init__.py:188]: May want something narrower than a bare `except` here?  At least `except Exception`?

**Comment by demianbrecht** [comment]
@bgrant all look good?

**Review by bgrant — COMMENTED** [review]
Looks good to me- I'm attempting to pull this version into a consuming codebase and see if any of the tests break.

**Review by bgrant — APPROVED** [review]
👍

**Pull request merged**